### ISSUE TITLE
BBC: 5.39.8 breaks CFAERBER/Graphics-ColorNames-WWW-1.14.tar.gz

### ISSUE THREAD

**Issue by andk** [Perl/perl5#22048]
**Description**

With v5.39.7-198-g351de214d2 tests started to fail for CFAERBER/Graphics-ColorNames-WWW-1.14.tar.gz

Sample fail report: http://www.cpantesters.org/cpan/report/97d1b22a-d4e5-11ee-be14-5b3b3320d34f

Bisect says:

``` text
351de214d258a4953da5633b4faa6b77e00f68e2 is the first bad commit
commit 351de214d258a4953da5633b4faa6b77e00f68e2
Author: Paul "LeoNerd" Evans <leonerd@leonerd.org.uk>
Date:   Wed Feb 7 17:19:26 2024 +0000

    Issue deprecation warnings on subsequent use VERSION declarations
```

**Steps to Reproduce**

`cpan -i CFAERBER/Graphics-ColorNames-WWW-1.14.tar.gz`

**Expected behavior**

should make, test, install

**Perl configuration**

```
# perl -V output goes here
Summary of my perl5 (revision 5 version 39 subversion 8) configuration:
  Commit id: 78455e6e7fa235411fb2b931eda9d816b5ea880f
  Platform:
    osname=linux
    osvers=6.5.0-18-generic
    archname=x86_64-linux-thread-multi-ld
    uname='linux k93jammy 6.5.0-18-generic #18~22.04.1-ubuntu smp preempt_dynamic wed feb 7 11:40:03 utc 2 x86_64 x86_64 x86_64 gnulinux '
    config_args='-Dprefix=/home/sand/src/perl/repoperls/installed-perls/host/k93jammy/v5.39.8/322f -Dmyhostname=k93jammy -Dinstallusrbinperl=n -Uversiononly -Dusedevel -des -Ui_db -Dlibswanted=cl pthread socket inet nsl gdbm dbm malloc dl ld sun m crypt sec util c cposix posix ucb BSD gdbm_compat -Duseithreads -Duselongdouble -DEBUGGING=-g'
    hint=recommended
    useposix=true
    d_sigaction=define
    useithreads=define
    usemultiplicity=define
    use64bitint=define
    use64bitall=define
    uselongdouble=define
    usemymalloc=n
    default_inc_excludes_dot=define
  Compiler:
    cc='cc'
    ccflags ='-D_REENTRANT -D_GNU_SOURCE -fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64'
    optimize='-O2 -g'
    cppflags='-D_REENTRANT -D_GNU_SOURCE -fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include'
    ccversion=''
    gccversion='11.4.0'
    gccosandvers=''
    intsize=4
    longsize=8
    ptrsize=8
    doublesize=8
    byteorder=12345678
    doublekind=3
    d_longlong=define
    longlongsize=8
    d_longdbl=define
    longdblsize=16
    longdblkind=3
    ivtype='long'
    ivsize=8
    nvtype='long double'
    nvsize=16
    Off_t='off_t'
    lseeksize=8
    alignbytes=16
    prototype=define
  Linker and Libraries:
    ld='cc'
    ldflags =' -fstack-protector-strong -L/usr/local/lib'
    libpth=/usr/local/lib /usr/lib/x86_64-linux-gnu /usr/lib /usr/lib64
    libs=-lpthread -lnsl -ldl -lm -lcrypt -lutil -lc
    perllibs=-lpthread -lnsl -ldl -lm -lcrypt -lutil -lc
    libc=/lib/x86_64-linux-gnu/libc.so.6
    so=so
    useshrplib=false
    libperl=libperl.a
    gnulibc_version='2.35'
  Dynamic Linking:
    dlsrc=dl_dlopen.xs
    dlext=so
    d_dlsymun=undef
    ccdlflags='-Wl,-E'
    cccdlflags='-fPIC'
    lddlflags='-shared -O2 -g -L/usr/local/lib -fstack-protector-strong'


Characteristics of this binary (from libperl): 
  Compile-time options:
    HAS_LONG_DOUBLE
    HAS_STRTOLD
    HAS_TIMES
    MULTIPLICITY
    PERLIO_LAYERS
    PERL_COPY_ON_WRITE
    PERL_DONT_CREATE_GVSV
    PERL_HASH_FUNC_SIPHASH13
    PERL_HASH_USE_SBOX32
    PERL_MALLOC_WRAP
    PERL_OP_PARENT
    PERL_PRESERVE_IVUV
    PERL_USE_DEVEL
    PERL_USE_SAFE_PUTENV
    USE_64_BIT_ALL
    USE_64_BIT_INT
    USE_ITHREADS
    USE_LARGE_FILES
    USE_LOCALE
    USE_LOCALE_COLLATE
    USE_LOCALE_CTYPE
    USE_LOCALE_NUMERIC
    USE_LOCALE_TIME
    USE_LONG_DOUBLE
    USE_PERLIO
    USE_PERL_ATOF
    USE_REENTRANT_API
    USE_THREAD_SAFE_LOCALE
  Built under linux
  Compiled at Feb 24 2024 05:29:46
  %ENV:
    PERL5LIB=""
    PERL5OPT=""
    PERL5_CPANPLUS_IS_RUNNING="1973"
    PERL5_CPAN_IS_RUNNING="1973"
    PERL_CANARY_STABILITY_NOPROMPT="1"
    PERL_MM_USE_DEFAULT="1"
    PERL_USE_UNSAFE_INC="1"
  @INC:
    /home/sand/src/perl/repoperls/installed-perls/host/k93jammy/v5.39.8/322f/lib/site_perl/5.39.8/x86_64-linux-thread-multi-ld
    /home/sand/src/perl/repoperls/installed-perls/host/k93jammy/v5.39.8/322f/lib/site_perl/5.39.8
    /home/sand/src/perl/repoperls/installed-perls/host/k93jammy/v5.39.8/322f/lib/5.39.8/x86_64-linux-thread-multi-ld
    /home/sand/src/perl/repoperls/installed-perls/host/k93jammy/v5.39.8/322f/lib/5.39.8
    .
```

**Comment by haarg** [comment]
This appears to be a problem in Graphics-ColorNames, not Graphics-ColorNames-WWW, and I believe should already be fixed by 8f770826b6c32799ae766feb28830018349c2d70.

**Comment by andk** [comment]
Thanks, I can confirm Graphics-ColorNames-WWW-1.14 passes tests with current bleadperl v5.39.8-34-g9233950dc7, closing ticket
